### PR TITLE
Merge subsystem branches

### PR DESCRIPTION
**Pull request by Speedy6451**
Monolith PR for new subsystems to be reviewed/finalized at programming meeting on 2/16

Confirmed to compile at [9cc482b927](https://github.com/Team-2502/RobotCode2023/commit/9cc482b92725bf44d9b67f25aad4902e2199aeeb)
Yet to be tested on-rio

### CODE REVIEW

**Review by Speedy6451 — COMMENTED** [review]
example comment

File: src/main/java/com/team2502/robot2023/Main.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  */
 public final class Main {
   private Main() {}
-
+//Test!!!
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/Main.java:16]: Can remove this

File: src/main/java/com/team2502/robot2023/subsystems/Manipulator.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.team2502.robot2023.subsystems;
+
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/subsystems/Manipulator.java:2]: Subsystem outdated

File: src/main/java/com/team2502/robot2023/subsystems/IntakeSubsystem.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        //TODO CHANGE SPEED!!!
+        liftMotor.setSmartCurrentLimit(20);
+        rightMotor.setSmartCurrentLimit(20);
+        leftMotor.setSmartCurrentLimit(20);
+
+        //Sets Left Motor To Spin Opposite Direction To Work With Right Motor
+        leftMotor.setInverted(true);
+    }
+
+    //Command to Deploy and Retract the Intake
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/subsystems/IntakeSubsystem.java:32]: Subsystem outdated

**Review by isaacmyers11 — APPROVED** [review]

**Review by lisat-projects — APPROVED** [review]

**Pull request merged**